### PR TITLE
Add Dag and Task IDs as env vars for SQL Statement annotation

### PR DESCRIPTION
**Pull request by rohang06**

### CODE REVIEW

File: airflow/models.py [hunk 1]
```diff
@@ -1396,7 +1399,10 @@ def signal_handler(signum, frame):
         except (Exception, KeyboardInterrupt) as e:
             self.handle_failure(e, test_mode, context)
             raise
-
+		finally:
+			# After execution, unset these env vars
+			os.environ.pop('CLOVER_DAG_ID', '')
+            os.environ.pop('CLOVER_TASK_ID', '')
```
Comment (wesleykendall) [inline airflow/models.py]: @rohang06 is this a typo with the tab spacing?

Comment (rohang06) [reply airflow/models.py]: Good catch, fixed in next diff

**Comment by wesleykendall** [comment]
This lgtm. @gwax do you have any suggestions on naming for this? And are you familiar with the other options we pursued for having globally-available information about the current airflow context?

(keep in mind this discussion is public)

**Review by gwax — CHANGES_REQUESTED** [review]
I'm pretty opposed to making Clover specific changes to Airflow as it will make long term maintenance and keeping up to date with upstream difficult.

Is there some other way we can do this?

**Comment by rohang06** [comment]
Closed in favor of the strategy we discussed offline

**Pull request closed without merging**